backend/src/routes/sets/sets.route.ts

```diff
@@ -0,0 +1,7 @@
+import express from "express";
+import { SetsController } from "../../controllers/sets/sets.controller";
+
+const router = express.Router({ mergeParams: true });
+router.get("/sets", SetsController.getSets);
```

Comment:
it would be better then to include some layer to keep routers in one place and in previous file (index.ts) only include routers, right not you have doubled this endpoint